lib/Config.js

```diff
@@ -1184,6 +1214,11 @@ class Config extends EventEmitter {
         };
     }
 
+    getPfsDaemonEndpoint(locationConstraint) {
+        return process.env[`${locationConstraint}_PFSD_ENDPOINT`] ||
+        this.locationConstraints[locationConstraint].details.pfsDaemonEndpoint;
```

Comment:
There should be a check for this new key in the getLocationConstraint function around line 330 :)